src/ui/patch_hub/patch_hub_core.sh

```diff
@@ -160,13 +160,13 @@ function show_registered_mailing_lists()
   message_box+='Select a mailing list to see the latest patchsets sent to it.'
 
   fallback_message='kw could not find any registered mailing list.'$'\n'$'\n'
-  fallback_message+='Try to register mailing lists in the menu "Settings" then ' 
+  fallback_message+='Try to register mailing lists in the menu "Settings" then '
   fallback_message+='"Register/Unregister Mailing Lists"'
 
   if [[ "${#registered_mailing_lists[@]}" -eq 0 ]]; then
-      create_message_box 'Error' "${fallback_message}"
-      screen_sequence['SHOW_SCREEN']='dashboard'
-      return "$?"
+    create_message_box 'Error' "${fallback_message}"
+    screen_sequence['SHOW_SCREEN']='dashboard'
+    return "$?"
```

Comment:
All these changes don't belong here, but in the third commit in the PR (the one where you introduced these lines). I see that you noticed the indentation problem I mentioned in the third commit, but this should be updated there using interactive rebase.